### PR TITLE
Log analytics SVT controls

### PR DESCRIPTION
**Pull request by SINIKI**
## Description
1. Added SVT controls for Log Analytics workspace
2. Updated SVT Mapping file

## Checklist
- [ ] I have read the instructions mentioned in [Contribute to Code](/Contributing.md).
- [ ] I have read and understood the criteria described under [submitting changes](/Contributing.md#submitting-changes). 
- [ ] The title of the PR clearly describes the intent of the PR.
- [ ] This PR does not introduce any breaking changes to the code.

### CODE REVIEW

File: src/AzSK/Framework/Core/SVT/Services/LogAnalytics.ps1 [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+			# If exception occur, control should go into Manual state
+			$controlResult.AddMessage([VerificationResult]::Manual, "Unable to get linked automation account for - [$($this.ResourceContext.ResourceName)].");
+		}
+		
+		return $controlResult;
+	}
+
+	hidden [ControlResult] CheckDataRetentionPeriod([ControlResult] $controlResult)
+	{
+		$controlResult.VerificationResult = [VerificationResult]::Verify;
```
Comment (v-zhgup) [inline src/AzSK/Framework/Core/SVT/Services/LogAnalytics.ps1]: Why we need this control, if its going to always be in verify state, unless we provide a safe retention period

File: src/AzSK/Framework/Core/SVT/Services/LogAnalytics.ps1 [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+			}
+		}
+
+		return $this.ResourceObject;
+	}
+
+	hidden [ControlResult] CheckResourceRBACAccess([ControlResult] $controlResult)
+	{	
+		$controlResult = $this.CheckRBACAccess($controlResult, $this.AccessList)
+		if ($this.ResourceObject.Properties.features.enableLogAccessUsingOnlyResourcePermissions -eq "false")
```
Comment (v-zhgup) [inline src/AzSK/Framework/Core/SVT/Services/LogAnalytics.ps1]: These nested properties are always available in resource object

File: src/AzSK/Framework/Core/SVT/Services/LogAnalytics.ps1 (hunk 1 above)
Comment (SINIKI) [reply src/AzSK/Framework/Core/SVT/Services/LogAnalytics.ps1]: Needs discussion. If we should add min retention period or remove this control completely.

File: src/AzSK/Framework/Configurations/SVT/Services/LogAnalytics.json [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+      },
+      {
+        "ControlID": "Azure_LogAnalytics_DP_Review_Monitoring_Solutions",
+        "Description": "Ensure that the monitoring solutions added to your log analytics workspace is published by a trusted provider",
+        "Id": "LogAnalytics140",
+        "ControlSeverity": "High",
+        "Automated": "Yes",
+        "MethodName": "CheckSolutionsLinkedToWorkspace",
+        "Rationale": "Monitoring solutions enables the author of the solution to capture sensitive data and execute custom code on your subscription. This can lead to a compromise of corporate data.",
+        "Recommendation": "Go to Azure Portal --> your Log Analytics workspace --> Solutions --> For each added solution: a. Check if the publisher is trusted b. Check the use of contained resources c. Check the use of referenced resources. For example, if your workspace is referencing a runbook, check if it is accessing any foriegn link, if it is performing any remote api call or executing any custom code (see detailed logs for the list of solutions).",
```
Comment (VishalHaibatpure) [inline src/AzSK/Framework/Configurations/SVT/Services/LogAnalytics.json]: Spell correction 
foriegn  --> foreign

File: src/AzSK/Framework/Configurations/SVT/Services/LogAnalytics.json [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+      },
+      {
+        "ControlID": "Azure_LogAnalytics_Audit_Configure_Data_Retention_Period",
+        "Description": "Data retention period should be configured for any security logs present in your workspace",
+        "Id": "LogAnalytics130",
+        "ControlSeverity": "Medium",
+        "Automated": "Yes",
+        "MethodName": "CheckDataRetentionPeriod",
+        "Rationale": "Data retention period enables you to perform effective auditing.",
+        "Recommendation": "Based on your requirement, you can choose to purge your data after a specificed duration. This duration/data retention period can be set from 31 days to 730 days. You can also specify different retention settings for individual data types. For effective logging, refer: https://docs.microsoft.com/en-us/azure/azure-monitor/platform/manage-cost-storage#change-the-data-retention-period",
```
Comment (VishalHaibatpure) [inline src/AzSK/Framework/Configurations/SVT/Services/LogAnalytics.json]: Spell correction
specificed --> specified

**Pull request merged**